native/java/org/jpype/javadoc/JavadocRenderer.java

```diff
@@ -0,0 +1,393 @@
+package org.jpype.javadoc;
+
+import java.nio.charset.StandardCharsets;
+import org.w3c.dom.Element;
+import org.w3c.dom.Node;
+import java.util.Map;
+import java.util.HashMap;
+
+/**
+ * Render a node as ReStructured Text.
+ *
+ * @author nelson85
+ */
+public class JavadocRenderer
+{
+
+  static final int WIDTH = 80;
```

Comment:
I guess 120 width is also fine nowadays. For instance jupyter notebooks render it with that, if I recall correctly.

Reply:
Change to 120 in final integration PR.